lib/entities/space.js

```diff
@@ -29,8 +29,14 @@ export function wrapSpace (http, data) {
   const spaceScopedHttpClient = wrapHttpClient(http.cloneWithNewParams({
     space: space.sys.id
   }), httpClientParams)
+  const spaceScopedUploadHttpClient = wrapHttpClient(http.cloneWithNewParams({
+    space: space.sys.id,
+    // host: 'upload.contentful.com'
```

Comment:
please avoid pushing commented properties, if it is for testing purpose try not to push them